lib/test.js

```diff
@@ -248,7 +248,11 @@ class Test {
 			return result;
 		};
 
+		const skipSnapshotHook = options.skipSnapshot || (() => {});
```

Comment:
"Hook" is a bit overloaded in AVA. How about `notifySkippingSnapshot` or something like that?

Reply:
`notifySkippingSnapshot` is definitely a better name. Creating this variable could also be avoided entirely, by checking whether `options.skipSnapshot` exists before calling it or by requiring that callers set it.

Reply:
The fewer variables the better 😉 May be tricky though since we don't initialize `Test` consistently across its tests.